modules/nf-lineage/src/main/nextflow/lineage/config/LineageConfig.groovy

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2013-2025, Seqera Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package nextflow.lineage.config
+
+import groovy.transform.CompileStatic
+import nextflow.Global
+import nextflow.Session
+
+/**
+ * Model workflow data lineage config
+ *
+ * @author Paolo Di Tommaso <paolo.ditommaso@gmail.com>
+ */
+@CompileStatic
+class LineageConfig {
+
+    final LineageStoreOpts store
```

Comment:
If the only option under this scope is `location`, then I think we can shorten it to `workflow.lineage.store`